File: area.c
#include <stdio.h>
#include <math.h>

int main() {
    int choice;
    printf("Choose a shape (1. Rectangle, 2. Circle, 3. Triangle): ");
    scanf("%d", &choice);
    
    switch (choice) {
        case 1:
            double length, width, rectangleArea;
            printf("Enter length and width of rectangle: ");
            scanf("%lf %lf", &length, &width);
            rectangleArea = length * width;
            printf("Area of rectangle: %lf\n", rectangleArea);
            break;
        case 2:
            double radius, circleArea;
            printf("Enter radius of circle: ");
            scanf("%lf", &radius);
            circleArea = M_PI * pow(radius, 2);
            printf("Area of circle: %lf\n", circleArea);
            break;
        case 3:
            double base, height, triangleArea;
            printf("Enter base and height of triangle: ");
            scanf("%lf %lf", &base, &height);
            triangleArea = 0.5 * base * height;
            printf("Area of triangle: %lf\n", triangleArea);
            break;
        default:
            printf("Invalid choice!\n");
    }
    
    return 0;
}

File: area.js
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Choose a shape (1. Rectangle, 2. Circle, 3. Triangle):");
        int choice = scanner.nextInt();
        
        switch (choice) {
            case 1:
                System.out.println("Enter length and width of rectangle:");
                double length = scanner.nextDouble();
                double width = scanner.nextDouble();
                double rectangleArea = length * width;
                System.out.println("Area of rectangle: " + rectangleArea);
                break;
            case 2:
                System.out.println("Enter radius of circle:");
                double radius = scanner.nextDouble();
                double circleArea = Math.PI * radius * radius;
                System.out.println("Area of circle: " + circleArea);
                break;
            case 3:
                System.out.println("Enter base and height of triangle:");
                double base = scanner.nextDouble();
                double height = scanner.nextDouble();
                double triangleArea = 0.5 * base * height;
                System.out.println("Area of triangle: " + triangleArea);
                break;
            default:
                System.out.println("Invalid choice!");
        }
        
        scanner.close();
    }
}
